### PR TITLE
Bump version, update about section

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,13 +26,17 @@ test:
     - clyent
 
 about:
-  home: https://github.com/Anaconda-Server/clyent
+  home: https://github.com/Anaconda-Platform/clyent
   license: BSD
```
Comment (jakirkham) [inline recipe/meta.yaml]: We should probably add license version here (e.g. `BSD 3-Clause`). Also is there a license file we can include?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-  name: clyent
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: clyent-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/c/clyent/clyent-{{ version }}.tar.gz
-  md5: a8a953a636cbd75021c3983f7ca60b5f
+  git_url: https://github.com/Anaconda-Server/{{ name }}.git
+  git_tag: {{ version }}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Why clone instead of downloading the source tarball?

File: recipe/meta.yaml (hunk 1 above)
Comment (nehaljwani) [reply recipe/meta.yaml]: They added a license file, but after the 1.2.2 tag.

File: recipe/meta.yaml (hunk 2 above)
Comment (nehaljwani) [reply recipe/meta.yaml]: 1.2.2 isn't available on pypi. :'( and it was released more than an year ago!

Comment (jakirkham) [reply recipe/meta.yaml]: Right, but we could use an archive from GitHub (e.g. https://github.com/Anaconda-Platform/clyent/archive/1.2.2.tar.gz ).

Comment (jakirkham) [reply recipe/meta.yaml]: Also raised issue ( https://github.com/Anaconda-Platform/clyent/issues/13 ) about getting a 1.2.2 release on PyPI.

**Comment by jakirkham** [comment]
Please update `conda-smithy` to 2.3.3 and then re-render.

File: recipe/meta.yaml (hunk 2 above)
Comment (nehaljwani) [reply recipe/meta.yaml]: I will update the recipe with that. But recently I faced a SHA256 mismatch with one such recipe using tarball from GH.

**Comment by nehaljwani** [comment]
The 're-add' already re-rendered it. Re-render with 2.3.3 reported no changes.

**Review by jakirkham — APPROVED** [review]

**Pull request merged**